### DISCUSSION TITLE
Is allowed to use a js function in Powershell?

### DISCUSSION THREAD

**Discussion by elenaromeo** [kjur/jsrsasign#580] [Q&A]
Hi,
I have a Postman collection that use that library and I need to convert this line code to Powershell:
Is that allowed?

I tried in Powershell with:
  $jsrsasignjs = Get-JsrsasignJs
  $sc = New-Object -ComObject ScriptControl;
  $sc.Language = "JavaScript";
  $sc.AddCode($jsrsasignjs );
but I get an error on the "$sc.AddCode($string)" ( "Expected identifier, string or number") even if the input is correct.

Is there another way to use it on Powershell?

Thank you

**Comment by kjur** [comment]
Sorry but jsrsasign doesn't support use in PowerShell.

**Reply by kjur** [reply]
I recommend to use PowerShell JWT library such as:
https://github.com/SP3269/posh-jwt
